src/test/kotlin/apply/domain/administrator/AdministratorTest.kt

```diff
@@ -0,0 +1,22 @@
+package apply.domain.administrator
+
+import apply.createAdministrator
+import io.kotest.assertions.assertSoftly
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class AdministratorTest : StringSpec({
+    "관리자명과 비밀번호를 수정한다" {
+        val administrator = createAdministrator("케이", "kth990303", "1234").apply {
+            update(
+                name = "케이케이",
+                password = "12345"
+            )
+        }
```

Comment:
r: 범위 지정 함수를 꼭 사용해야 할까요?

```suggestion
        val administrator = createAdministrator("케이", "kth990303", "1234")
        administrator.update(name = "케이케이", password = "12345")
```

Reply:
코멘트대로 변경했습니다!
다만 궁금한 점이, 어떠한 때에 scope function을 사용하고 어떠한 때에 사용하지 않는지 잘 모르겠습니다.

`ApplicationFormTest`에서 `지원서를 수정한다`의 테스트 코드는 scope function을 사용하지 않아도 됨에도 불구하고 scope function을 사용하여 update를 한 후에 assertSoftly로 테스트가 진행됐으며, `포트폴리오 없이 지원서를 수정한다`의 테스트 코드는 제이슨이 코멘트 남겨주신대로 scope function을 사용하지 않고 테스트가 진행됐더라구요.
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/57135043/194738781-db2762df-dd5c-483b-b97d-08a0506536df.png">

혹시 test code에서 scope function을 사용하는 기준이 따로 있는지 궁금합니다!